### DISCUSSION TITLE
Slow movement

### DISCUSSION THREAD

**Discussion by FakeForFearofFlame** [isid0re/SoloLeveling#55] [Q&A]
Bot runs fine for 1-5 minutes then starts walking incredibly slow. Any suggestions?

Thanks

**Comment by isid0re** [comment]
There is a known bug in kolbots core, where if the bot gets "frozen" kolbot doesnt release the frozen state. The only cure for that situation is for the bot to make a new game.

**Reply by toxickitteh** [reply]
is there a way for the bot to detect this and maybe rejoin the game? this is completely screwing with town routines....

**Reply by isid0re** [reply]
I wish I knew a way to do that. That would be great to automate a fix.  Unfortunately I dont.
